app/components/channel_drawer/channels_list/list/list.js

```diff
@@ -218,6 +227,25 @@ class List extends PureComponent {
         this.setState({width: width - 40});
     };
 
+    onPreview = (item) => {
+        if (Platform.OS === 'ios') {
+            const {intl} = this.context;
+
+            this.props.navigator.push({
+                screen: 'ChannelPeek',
+                previewCommit: false,
+                previewView: this.itemRefs[item],
+                previewActions: [{
```

Comment:
Is it possible to have this defined on the ChannelPeek component itself? It feels weird that its parent needs to know this

Reply:
Nope these actions are part of what the API expects to receive